i915/Makefile

```diff
@@ -12,11 +12,11 @@ SRCS=	\
 	i915_buddy.c \
 	i915_cmd_parser.c \
 	i915_drv.c \
-	i915_getparam.c \
```

Comment:
Does this refactoring adds any file ?
It's hard to see ...
Also some of the new ordering doesn't match what I have with emacs M-x sort-lines which is usually what I use, so I don't like that.
If any files are missing, please add them and just that.

Reply:
It does add a few files. I have reverted to the old Makefile and removed the respective commits (sorry for force pushing), and only added the needed files.